shared/constants/platform.desktop.js

```diff
@@ -56,10 +56,19 @@ function win32SocketDialPath (): string {
 }
 
 function linuxSocketDialPath (): string {
-  if (runMode === 'prod') {
-    return path.join(`${getenv('XDG_RUNTIME_DIR', '')}/keybase/`, socketName)
+  // If XDG_RUNTIME_DIR is defined use that, else use $HOME/.config.
+  const homeDir = getenv('HOME', '')
+  const homeCacheDir = path.join(homeDir, '.config')
```

Comment:
Nit: the cache dir makes it sound like this variable refers to `~/.cache`. Maybe call it `homeConfigDir`?